vue-app/src/api/core.ts

```diff
@@ -2,6 +2,9 @@ import { ethers } from 'ethers'
 
 import { FundingRoundFactory } from './abi'
 
+export const mainnetProvider = new ethers.providers.StaticJsonRpcProvider(
+  process.env.VUE_APP_MAINNET_PROVIDER_URL
```

Comment:
```suggestion
  process.env. VUE_APP_ETHEREUM_MAINNET_API_URL
```
I suggest sticking with the "API" terminology for consistency.

Reply:
Also please add this ENV variable to the .env.example file.